### PR TITLE
[Suggestion] Change formatting of /gtfo in-game

### PR DESCRIPTION
**Pull request by TheCool1Mike**
Right now, the admin's name is automatically stated when the banned player tries to rejoin. As a result, we can't find who is banning a player (without going on logviewer) as we don't put our signature anymore.

### CODE REVIEW

File: src/main/java/me/totalfreedom/totalfreedommod/commands/Command_gtfo.java [hunk 1]
```diff
@@ -79,6 +79,8 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
         // Broadcast
         final StringBuilder bcast = new StringBuilder()
                 .append(ChatColor.RED)
+                .append("sender.getName()")
```
Comment (DUFCLiam) [inline src/main/java/me/totalfreedom/totalfreedommod/commands/Command_gtfo.java]: Error on line 82 "sender.getName()" should be `.append(sender.getName())`

**Comment by DUFCLiam** [comment]
The branches conflict. Please use the TFM 5.0 branch :)

**Comment by LegendIsAwesomes** [comment]
Associated issue?

**Comment by TheCool1Mike** [comment]
Going to fix conflicts.

/Closed/

**Comment by TheCool1Mike** [comment]
See #1556

**Comment by JeromSar** [comment]
@TheCool1Mike **Please do not spam PRs on this repository**. You can fix conflicts in the same PR. You do not have to close any PRs whatsoever.

**Pull request closed without merging**